### DISCUSSION TITLE
www.nikse.dk down

### DISCUSSION THREAD

**Discussion by Escape888** [SubtitleEdit/subtitleedit#8267] [General]
surfing to www.nikse.dk leads to a **HTTP Error 503. The service is unavailable.**

Is this a temporary of permanent issue?

**Comment by niksedk** [comment]
Hm, the web hotel support says nikse.dk has used too many resources, but they should have unlimited bandwidth, so that sounds weird.

So I do not know how this will end...

**Reply by GenSF** [reply]
It went back online for a day or two but now is leads to the same error...

**Comment by niksedk** [comment]
Hm, it seems hard to get the web site up and running again... I will start moving the documentation to GitHub.

**Comment by niksedk** [comment]
Nikse.dk is up again, with more cpu and memory - and with some memory leaks fixed :)

If you wish to support the web site, do donate: https://www.nikse.dk/donate

**Reply by Hermit6202** [reply]
Sorry, down again. 
"Service Unavailable
HTTP Error 503. The service is unavailable."

**Reply by Jds1234** [reply]
yes, it is still down with 503 error, as of 24/05/2024. according to... https://www.isitdownrightnow.com/nikse.dk.html ...says its currently under maintenence.

**Comment by sherkhan30452** [comment]
I think it is safe to close this old discussion since it's working now. Unless @Jds1234 is still unable to reach this website.

**Reply by Jds1234** [reply]
yes it is working now, thanks.  :)

**Reply by sherkhan30452** [reply]
@Jds1234 Please close this discussion, thank you! :)

**Comment by Escape888** [comment]
Yes, close, this has been resolved a while ago now.

**Reply by Jds1234** [reply]
hmmm, funny... no close discussion option for me here, seems you started the discussion @Escape888, i think its your option to close it.

look at the top, it says "Escape888 started this conversation in General"

**Comment by Escape888** [comment]
This issue has been resolved and closed.

**Comment by niksedk** [comment]
I'm afraid that the web site is somewhat unstable... I will move the SE help pages when I have time.

**Comment by GenSF** [comment]
I'm a big fan of the subtitle editor online, it's way more simple and has all the tools i need , and being flash based format i can put the subtitles with more precision by dragging them to where I want. I can't do that on the main program awhile the video runs at the same time. But lately often appears the 'service unavailable '. Is there anything i can do to work on the main program like i work with the online?

**Comment by ananaimous** [comment]
Hi, looks like nikse.dk is down again. I had important srt files in the process of editing and at some point I couldn't download it from the website, only to find the 503 not available message. Will I get my supposedly 'backed up' files after the online tool is fixed?

**Comment by A-Taci** [comment]
When will this problem be solved?
Is it possible to start our own WebService?

**Comment by aCrazyPOTATO** [comment]
Anyone got the assets from https://[www.nikse.dk](https://www.nikse.dk/assets/SubtitleEdit/SeNetworkingServiceCore.zip)/assets/SubtitleEdit/SeNetworkingServiceCore.zip 
The Website is always down.
